addons/payment_stripe_checkout_webhook/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+{
+    'name': 'Stripe Payment Acquirer',
+    'summary': 'Payment Acquirer: Stripe Implementation',
+    'version': '1.0',
+    'description': """
+Stripe Payment Acquirer checkout webhook
+========================================
+    
+Allow configuring a webhook in Stripe to send s2s notifications to Odoo
+when a Checkout payment is completed. Note that SetupIntent and
+PaymentIntent events are not listened to, since they are handled 'live'
+with the customer actively present; the main use case for Stripe
+webhooks is a Checkout session that gets interrupted before the customer
+is redirected to Odoo (e.g. network loss, browser crash, closing the
+tab, etc.).
+
+The webhook should be configured to send its events to
+<base_url>/payment/stripe/webhook and should only subscribe to
+checkout.session.completed events to avoid spamming the Odoo server with
+useless notifications.""",
+    'depends': ['payment', 'payment_stripe', 'payment_stripe_sca'],
```

Comment:
```suggestion
    'depends': ['payment_stripe', 'payment_stripe_sca'],
```
`payment_stripe` already depends on `payment` but it's not a big deal in this case.

Reply:
> Side note: I think that you should set yourself as the author of the commit since it's not an exact backport of [dc4f6ad](https://github.com/odoo/odoo/commit/dc4f6ad04f8bf3800c650d878b08e44775b95628).

it is being backported by aho, sig created the PR for his work:

https://www.odoo.com/web?debug#action=327&cids=1&id=2449738&menu_id=4720&model=project.task&view_type=form